aws/index.js

```diff
@@ -0,0 +1,12 @@
+import aws from "aws-sdk";
+import nodemailer from "nodemailer";
+
+// configure AWS SDK
+const ses = new aws.SES({
+    apiVersion: "2010-12-01",
+});
```

Comment:
we need to configure the AWS credentials as well for it to work when deployed

Reply:
Perhaps we could have something like how it was done in dancefest in this file:

![image](https://user-images.githubusercontent.com/83791196/121959967-555e0180-cd1a-11eb-960f-5384994efbaa.png)

Reply:
What do you mean by configure? Shouldn't the deployed version have environment variables for AWS_KEY, etc.?

Reply:
You can do something like this:
![image](https://user-images.githubusercontent.com/46411197/121980839-843b9e00-cd41-11eb-8ca4-6ae6fb9fbb01.png)

Reply:
OK.